public/js/selfoss-events-navigation.js

```diff
@@ -71,7 +71,7 @@ selfoss.events.navigation = function() {
         $(this).addClass('active');
         
         if($(this).hasClass('nav-tags-all')==false) {
-            location.hash = '#' + selfoss.filter.type + '/tag-' + $(this).find('span').html();
+            location.hash = '#' + selfoss.filter.type + '/tag-' + $(this).find('span').html().replace('%', '%25');
```

Comment:
Yes, but I had preferred the browsers to show a unencoded string in the address bar. I've also tested with both browers.